feg/gateway/services/n7_proxy/servicers/session_controller.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/clientcredentials"
+
+	fegprotos "magma/feg/cloud/go/protos"
+	"magma/feg/gateway/policydb"
+	n7_client "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
+	"magma/feg/gateway/services/n7_proxy/metrics"
+	"magma/lte/cloud/go/protos"
+	orcprotos "magma/orc8r/lib/go/protos"
+)
+
+type CentralSessionController struct {
+	policyClient  *n7_client.ClientWithResponses
+	dbClient      policydb.PolicyDBClient
+	cfg           *SessionControllerConfig
+	healthTracker *metrics.SessionHealthTracker
+}
+
+type SessionControllerConfig struct {
+	PCFConfig      *PCFConfig
+	RequestTimeout time.Duration
+	DisableN7      bool
```

Comment:
If you check session_proxy, this  field comes from mconfig field with the same name, 
This field it is noot related with Enabled/Disabled status on health service